api/routes/testAPI.js

```diff
@@ -8,7 +8,7 @@ const bodyParser = require('body-parser');
 const app = express();
 app.use(bodyParser.json());
 
-// Create a Sequelize instance and connect to the database
+
 const sequelize = new Sequelize('mynew', 'root', '', {
```

Comment:
add this 'mynew' , 'root' all this confidential information in .env file and add the variable names here... This confidential info should not kept in open